### ISSUE TITLE
CoreOS docker daemon not restarted after reboot

### ISSUE THREAD

**Issue by pdion891** [cloud-ca/docker-machine-driver-cloudca#4]
When CoreOS is deployed using docker-machine-driver-cloudca, after an automatic upgrade of CoreOS, docker daemon is not restarted by default, default behavior of CoreOS. 

Maybe we should push  the command via cloud-init

```echo "docker ps" >> /etc/rc.local```

as describe in https://github.com/rancher/rancher/issues/5069

**Comment by franzpgarcia** [comment]
Is this fixed by #3?

**Comment by pdion891** [comment]
No, #3 only make datavolume a persistent storage and not wiped after a reboot. 

I cannot fix #4 with #3 because if a user do not user data-volume he is stuck with this bug.
